src/libraries/System.Security.Cryptography.Pkcs/tests/SignedCms/SignedCmsTests.netcoreapp.cs

```diff
@@ -471,6 +566,32 @@ public static void AddSigner_ECDSA_EphemeralKey()
             }
         }
 
+        [ConditionalFact(typeof(SlhDsa), nameof(SlhDsa.IsSupported))]
+        public static void AddSigner_SlhDsa_EphemeralKey()
+        {
+            // TODO This key is imported so is it ephemeral?
+            using (SlhDsa slhDsa = SlhDsa.ImportSlhDsaSecretKey(SlhDsaAlgorithm.SlhDsaSha2_128s, SlhDsaTestData.IetfSlhDsaSha2_128sPrivateKeyValue))
+            using (X509Certificate2 publicCertificate = Certificates.SlhDsaSha2_128s_Ietf.GetCertificate())
+            using (X509Certificate2 certificateWithKey = Certificates.SlhDsaSha2_128s_Ietf.TryGetCertificateWithPrivateKey(exportable: true))
+            {
+                if (certificateWithKey == null)
+                {
+                    return;
+                }
```

Comment:
I'm not sure this is important here.  I believe in the other tests it's "If this cert doesn't have a known private key".  Since you know your private key, I don't know how this would come up.